ui/src/shared/constants/colorOperations.js

```diff
@@ -46,12 +46,16 @@ export const stringifyColorValues = colors => {
   return colors.map(color => ({...color, value: `${color.value}`}))
 }
 
-export const generateThresholdsListHexs = (
+export const generateThresholdsListHexs = ({
   colors,
   lastValue,
-  containsLineGraph
-) => {
-  const defaultColoring = {bgColor: null, textColor: THRESHOLD_COLORS[11].hex}
+  containsLineGraph,
+  cellType,
```

Comment:
maybe containsLineGraph and cellType should have defaults, since they are optinal?